templates/common/components/signalement-detail-evenements.html.twig

```diff
@@ -26,9 +26,17 @@
                     <a href="#" class="{{ event.actionLink }}">{{ event.actionLabel }}</a>
                 </div>
             {% else  %}
-                <div class="event-item-action fr-mt-3v">
-                    <a href="{{ event.actionLink }}" {% if event.domain is not defined %} target="_blank" {% endif %}>{{ event.actionLabel }}</a>
-                </div>
+                {% if event.domain is same as(constant('App\\Entity\\Event::DOMAIN_PROTOCOLE') ) %}
+                    <div class="event-item-action fr-mt-3v">
+                        <a class="fr-link fr-link--download" download href="{{ event.actionLink }}">
+                            {{ event.actionLabel }}<span class="fr-link__detail">PDF</span>
```

Comment:
Ici je n'ai pas suivi tout à fait les recommandations du DSFR car cela aurait nécessité une migration de la table event pour stocker la taille du pdf à part

Reply:
Pourquoi stocker la taille à part ? On l'a pas mis dans de la conf ? Je crois qu'on s'en sert ailleurs !

Reply:
Oui il y a les tailles des pdf dans la conf, mais je ne sais pas trop comment identifier le bon pdf vu que tout est générique pour l'affichage des évenements...
il faudrait que je compare le lien vers le pdf à `$this->getParameter('doc_autotraitement')` et $this->getParameter('doc_domicile') pour savoir si je dois utiliser  `$this->getParameter('doc_autotraitement_size')` ou `$this->getParameter('doc_domicile_size')` dans le `span class="fr-link__detail"`, sachant que la taille est déjà indiquée dans `event.actionLabel` ça me semble beaucoup de complication pour juste l'afficher dans le bon span, non ?

Reply:
En effet, le mieux est l'ennemi du bien
Mais peut-être mettre une taille approximative ? genre 1Mo ? et pas grave si c'est pas exact ? ou la moyenne entre les deux ?

Reply:
En dur du coup ?

Reply:
En fait, le seul téléchargement qu'on affiche dans les évenements, c'est pour l'autotraitement, et on a sa taille plus haut dans le twig. Donc je l'affiche bien au bon endroit, et je supprime la taille du label de l'événement